ingest/author_de.py

```diff
@@ -276,11 +282,12 @@ def check_group(names_dict, name):
 def sort_comparison(ls):
     """
     Naturally sort groups in a pairwise comparison; specially handle one-vs-rest
-    this should take in only a list of two, ex (type1 type2)
+    this should take in a combined name, such as B cells--CSN1S1 macrophages
```

Comment:
The usage reference passes in a list of groups, and the implementation here also operates on a list:

```suggestion
    this should take in a list of groups, such as ['B cells', 'CSN1S1 macrophages']
```